drivers/sensor/isl29125/Kconfig

```diff
@@ -0,0 +1,52 @@
+# ISL29125 Color Light to Digital Converter configuration options
+
+# Copyright(c) 2020 Intive
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ISL29125
+	bool "ISL29125 Color Light to Digital Converter"
+	depends on I2C
+	help
+		Enable driver for ISL29125 Color Light to Digital Converter.
+
+if ISL29125
+
+choice
+	prompt "Trigger mode"
+	depends on GPIO
+	default ISL29125_TRIGGER_NONE
+	help
+		Specify the type of triggering to be used by the driver.
+
+config ISL29125_TRIGGER_NONE
+	bool "No trigger"
+
+config ISL29125_TRIGGER_GLOBAL_THREAD
+	bool "Use global thread"
+	depends on GPIO
+	select ISL29125_TRIGGER
+
+config ISL29125_TRIGGER_OWN_THREAD
+	bool "Use own thread"
+	depends on GPIO
+	select ISL29125_TRIGGER
+endchoice
+
+config ISL29125_TRIGGER
+	bool
+
+config ISL29125_THREAD_PRIORITY
+	int "Thread priority"
+	depends on ISL29125_TRIGGER_OWN_THREAD
+	default 10
+	help
+	Priority of thread used by the driver to handle interrupts.
+
+config ISL29125_THREAD_STACK_SIZE
+	int "Thread stack size"
+	depends on ISL29125_TRIGGER_OWN_THREAD
+	default 1024
+	help
+	Stack size of thread used by the driver to handle interrupts.
```

Comment:
This line should be indented two space more than the previous line. (i.e., one tab and two spaces)